files/en-us/web/css/anchor/index.md

```diff
@@ -319,5 +313,8 @@ Try selecting different values from the drop-down menus to see how they affect t
 
 ## See also
 
-- [CSS anchor positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning)
+- {{cssxref("position-anchor")}}
+- {{cssxref("inset-area")}}
 - [Using CSS anchor positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning/Using)
+- [Handling overflow: try options and conditional hiding](/en-US/docs/Web/CSS/CSS_anchor_positioning/Try_options_hiding)
+- [CSS anchor positioning](/en-US/docs/Web/CSS/CSS_anchor_positioning) module
```

Comment:
please add related features on pages as appropriate.
Inlcude "module" after each module link in see alsos
add the handling overflow as appropriate (not 100% sure it's needed here, but doesn't hurt). Its not in many pages since it was created after the PRs were submitted

Reply:
I'll update the see also links as appropriate on each feature page.